main/guides/coreeval/local-testnet.md

```diff
@@ -2,26 +2,44 @@
 
 To create, start, and deploy an Agoric governance proposal to a local Agoric Testnet, do the following:
 
-1. Navigate to `<agoric-sdk>/packages/cosmic-swingset` and run `make`.
-2. Prepare the chain and solo nodes.
+1. If you're using a Mac, ensure that you have [Xcode](https://apps.apple.com/us/app/xcode/id497799835) installed.
+2. Create a project folder that will allow you to run a local blockchain:
 
 	```jsx
-	make scenario2-setup BASE_PORT=8000 NUM_SOLOS=0
+	agoric init <The name of your project>
 	```
-3. Start the chain.
+    
+    For example, if you wanted to run your local blockchain from a folder named "Demo", you'd run this command:
+    
+    ```jsx
+    agoric init Demo
+    ```
+
+	**Note:** Your project folder should *not* be located within your local clone of the agoric SDK.
+
+3. Install additional dependencies by entering your project folder and running the following command.
+
+	```jsx
+    cd <Name of your project (e.g., Demo)>
+    agoric install
+    ```
+
+4.  Start the chain by running the following command. (**Note:** You should still be located within your project folder.)
 
 	```jsx
-	make scenario2-run-chain
+	agoric start local-chain --verbose --reset
 	```
-4. Switch to another terminal and wait for the first block to be produced.
-5. Navigate to `<agoric-sdk>/bin` and submit the governance proposal.
+
+5. Wait for the first block to be produced.
```

Comment:
"How can I tell?" I can imagine our audience asking. I suggest providing an example log entry showing a block being produced.

(suggestion; not critical / important)